sample_project/input_config.yaml

```diff
@@ -3,3 +3,10 @@ metadata_fields_file_path: ./sample_project/metadata_fields.yaml
 metadata_key_field_str: File
 pose_estimation_model_str: DLC_resnet50_jwasp_femaleandmaleSep12shuffle1_1000000
 pose_estimation_results_path: ./sample_project/pose_estimation_results
+roi_names:
+  - "enclosure"
+  - "left_box"
+  - "top_box"
+  - "right_box"
+  - "bottom_box"
+  - "tube"
```

Comment:
Somewhere or somehow we should flag up that one simply needs to add a row here to define a new RoI.

##### Minor thought:
_Do we want a button for this? ... I have a feeling you've discussed it already. Feels like it's very much a: **define your config once, at the start**, kind of vibe. We would then get into annoying edge cases where users added a new RoI name halfway through the RoI definitions and the metadata would become inconsistent between videos._

Reply:
Hmm, interesting thought. 
We could even think bigger: we could have the user define the config in the "Home" tab in a graphical way.
I am thinking of sth  like a form:
* Select your video folder
* Select your model folder
* Choose your ROIs, etc.

This would then become the canonical way in which a user configures a project. Modifications at a later stage (i.e. halfway through analysis) are possible, but we would have to handle them in a clever way: allowing some, and throwing warnings/exceptions when a config change threatens the project integrity. Nevertheless, all of this is sth better left for v1. Should I raise a ticket?

Reply:
Oh, right: I do like this. Certainly I like it much better than my "add RoI" button which has the potential to misallign the metadata between videos.

Your idea is:
  * I startup WAZP and either 
       * load a config file, 
       * or click some buttons to set up my project.
   * Then move on to metadata → RoIs.

This feels like a decent `new feature` ticket for v1. Perhaps we get some feedback from the researchers? It's also possible that editing a simple yaml config file like this is OK.

Reply:
Yeah, that's precisely the idea. But you are right that it could be overengineering at this point. I will raise an issue, to remember it, but will leave it as potential for now (till feedback)

Reply:
Now the above is described in [issue 43](https://github.com/SainsburyWellcomeCentre/WAZP/issues/43)